large_image/report.py

```diff
@@ -0,0 +1,65 @@
+import scooby
+
+
+class Report(scooby.Report):
```

Comment:
It'd be best if this could show some usage information here (I think adding `scooby.Report.__doc__` as the doc string of the class would work).